Integrations/integration-SplunkPy.yml

```diff
@@ -187,5 +212,25 @@ script:
       default: true
       description: search id
     description: 'fetch splunk search results '
+  - name: splunk-search
+    arguments:
+    - name: query
+      required: true
+      default: true
+      description: The splunk search language string to execute, example :"* | head
+        3"
+    - name: earliest_time
+      description: Specifies the earliest time in the time range to search. The time
+        string can be a UTC time (with fractional seconds), a relative time specifier
+        (to now), or a formatted time string. Default is 1 week ago
```

Comment:
I think example should be more clear (e.g. "2 days ago") 
also - tried to give `earliest_time="1 week ago"` and it does not work